nemesis/nemesis_pathing.py

```diff
@@ -865,7 +865,7 @@ def calm_lands_1_old(checkpoint):
 
 
 def calm_lands_1(checkpoint):
-    # print("CKP:", checkpoint)
+    # logger.debug("CKP:", checkpoint)
```

Comment:
For all the commented out `logger.debug()` in this that have checkpoint parameters, they need to be f-strings.

Reply:
Should be removed now. Some of these commented prints/logs were like 'maybe I'll need these later' and then never did.